fdbclient/BulkLoading.cpp

```diff
@@ -181,7 +181,7 @@ BulkLoadTaskState createBulkLoadTask(const UID& jobId,
 	    fileSet, range.begin, range.end, snapshotVersion, bytes, keyCount, byteSampleSetting, type, transportMethod);
 	BulkLoadManifestSet manifests(1);
 	manifests.addManifest(manifest);
-	return BulkLoadTaskState(jobId, manifests);
+	return BulkLoadTaskState(jobId, manifests, range);
```

Comment:
Didn't we use to specify the range when we ran a bulkdump in the past? We had to provide "" \xff.  Is it that this range was ignored but now you are adding the range feature?

Reply:
Currently, bulkdump is indeed supporting the range dump. However, in the simulation test, we only set the range to `"" \xff`. I will add a test for per range dump.